tests/app/views/contexts/summary/test_question.py

```diff
@@ -216,14 +306,107 @@ def test_concatenate_textfield_answers(self):
                     schema=self.schema,
                     rule_evaluator=self.get_rule_evaluator(),
                     location=None,
-                    block_id="house-type",
+                    block_id="address-block",
+                    return_to=None,
+                )
+
+                # Then
+                self.assertEqual(
+                    question.answers[0]["value"],
+                    f"Government Buildings{concatenation_character}Cardiff Rd{concatenation_character}Newport{concatenation_character}NP10 8XG",
+                )
+                self.assertEqual(len(question.answers), 1)
+
+    def test_concatenate_textfield_answers_default(self):
+        answer_separators = {"Newline": "<br>", "Space": " "}
```

Comment:
A bit of duplication in this file now with the QuestionnaireSchema and the 2 tests having sharing all the same chunks 
```
{
                    "id": "building",
                    "label": "Building",
                    "mandatory": False,
                    "type": "TextField",
                    "default": "Government Buildings",
                }
```

Reply:
Repetition removed now, no repeating chunks!